app/gui2/tsconfig.app.json

```diff
@@ -1,12 +1,27 @@
 {
   "extends": "@vue/tsconfig/tsconfig.dom.json",
-  "include": ["env.d.ts", "src/**/*", "src/**/*.vue", "shared/**/*"],
+  "include": [
+    "env.d.ts",
+    "src/**/*",
+    "src/**/*.vue",
+    "shared/**/*",
+    "shared/**/*.vue",
+    "public/**/*",
+    "public/**/*.vue"
+  ],
   "exclude": ["src/**/__tests__/*"],
   "compilerOptions": {
+    "lib": ["ES2021", "DOM", "DOM.Iterable"],
     "composite": true,
     "baseUrl": ".",
+    "allowImportingTsExtensions": true,
+    "strict": true,
```

Comment:
`strict` is already present in extended config. Why do we need `allowImportingTsExtensions`? I see it's used in visualizations, but not sure why. Can't it work without the `.ts` suffix?

Reply:
it's required in visualizations to be able to find the import, since the source files do have a `.ts` extension. it's not necessary if they're moved into `src/` though